tests/installation/bootloader_s390.pm

```diff
@@ -92,7 +92,7 @@ sub prepare_parmfile {
         if (get_var('AGAMA')) {
             $params .= " root=live:ftp://" . get_var('REPO_HOST', 'openqa') . '/' .
               ((get_var('FLAVOR') eq "Full") ?
-                  get_required_var('REPO_0') . "/LiveOS/squashfs.img" :
+                  get_required_var('REPO_0') . "/LiveOS/squashfs.img"  . " live.password=$testapi::password" :
```

Comment:
here wasn't in the first place, so probably is added via our settings in job group.

Reply:
thanks for review, I removed it from this .pm file.